src/cmd.rs

```diff
@@ -7,24 +7,18 @@ use std::{
 use crate::StdError;
 
 pub fn run(args: &str, shell: Option<&str>) -> Result<(String, String, ExitStatus), StdError> {
-    let output =
-        fake_tty::command(&format!("{}; printf \"~~////~~\"; pwd", args), shell)?.output()?;
+    let cwd_before = std::env::current_dir()?;
+
+    let output = fake_tty::command(args, shell)?.output()?;
 
     let stdout = fake_tty::get_stdout(output.stdout)?;
     let stderr = String::from_utf8(output.stderr)?;
     let status = output.status;
 
-    let stdout = stdout.trim_end();
-    let lb = stdout
-        .rfind("~~////~~")
-        .ok_or_else(|| format!("Delimiter not found in the string {:?}", stdout))
-        .unwrap();
-
-    let (output, cwd) = stdout.split_at(lb);
-    let cwd = cwd.trim_start_matches("~~////~~").trim_start();
+    let output = stdout.trim_end();
 
-    if !cmp_paths(std::env::current_dir()?, cwd) {
-        std::env::set_current_dir(cwd)?;
+    if !cmp_paths(std::env::current_dir()?, &cwd_before) {
+        std::env::set_current_dir(cwd_before)?;
```

Comment:
That's not what the original code did. When you do

```sh
to-html "cd .." "ls"
```

The 1st command changes directory, and the 2nd command should print the entries in that directory. The problem was that running `cd` in `fake_tty::command` does not affect the working directory of the current process, so I wrote this "hack".

Reply:
Expressed this behavior in a test with this commit (https://github.com/Aloso/to-html/commit/e66a380b2412bc496d6d2a1a9034a0fa1247551e). It indeed succeeds on `main`, but fails with this PR

Reply:
Whelp that's odd. That change appears to fail consistently on only the ubuntu, rustc beta job with output that looks identical to not actually changing the output. I'm not able to reproduce it locally when running the `1.88.0-beta.3` toolchain on Arch :/

Reply:
ah, looks like I've never changed directories when using `to-html`!

I've fixed the directory changing code -- getting it to work on Windows is a little hacky (`std::env::set_current_dir` doesn't like `pwd` outputs, which are different depending on which `bash` you're running).

@CosmicHorrorDev I've taken that commit and added it to this branch (and used `ls -1` to avoid the `\t` issue).